Content/NPCs/Vitric/MagmitePassive.cs

```diff
@@ -10,6 +10,11 @@ namespace StarlightRiver.Content.NPCs.Vitric
 {
 	internal class MagmitePassive : ModNPC
 	{
+		private bool isBestiaryEntry = true; // defaults to true, is set to false on spawn (as all real NPCs run OnSpawn)
```

Comment:
What exactly is the purpose of this, NPCs already have a flag for if they are a bestiary entry, no?

Reply:
You are absolutely correct, I committed a dumb.